app/covidbed/model/orga.py

```diff
@@ -45,45 +48,49 @@ class Address(Base):
 
 
 class Region(Base):
-    __tablename__ = 'orga_region'
+    __tablename__ = "orga_region"
 
-    code = db.Column(db.String(2),  primary_key=True)
+    code = db.Column(db.String(2), primary_key=True)
     tncc = db.Column(db.String(20), nullable=False)
-    libelle = db.Column(db.String(30), nullable=False)
+    libelle = db.Column(db.String(31), nullable=False)
```

Comment:
Has the migration been added ?

Reply:
Sorry, probably just a shortcut miss typed which incremented the integer, fixed